bitnami/cassandra/README.md

```diff
@@ -92,8 +92,9 @@ The following tables lists the configurable parameters of the cassandra chart an
 | `cluster.rack`                       | Rack name                                                                                                                                                 | `rack1`                                                      |
 | `cluster.enableRPC`                  | Enable Thrift RPC endpoint                                                                                                                                | `true`                                                       |
 | `cluster.enableUDF'                  | Enable CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS                                                                                                            | `false`                                                      |
-| `cluster.pdbEnabled`                 | Enable the creation of a Pod Disruption Budget to ensure a minimun number of pods are running.                                                            |               `true`                                                      |
-| `cluster.minimumAvailable`           | Minimum number of instances that must be available in the cluster (used of PodDisruptionBudget) Needs `cluster.pdbEnabled=true`                           | `1`                                                          |
+| `cluster.pdbEnabled`                 | Enable the creation of a Pod Disruption Budget to ensure a minimun number of pods are running.                                                            | `true`                                                       |
+| `cluster.minAvailable`               | Minimum number of instances that must be available in the cluster (used of PodDisruptionBudget) Needs `cluster.pdbEnabled=true`                           | `nil`                                                        |
```

Comment:
Hi,
  Could you rename this value to keep the same name ?.
  This way there won't be changes for the users.

Reply:
Please notice that in go code this variable was named minAvailable. Less breaking change is fixing documentation rather than fixing go-lang code, no? :)

Reply:
Sorry, I think I not expressed my thought very well.
I wanted to mean to keep it the same with respect to previous versions. This way backwards compatibility is kept.
And yes, you'd have to change not only in this file.